nutils/function.py

```diff
@@ -2452,6 +2452,27 @@ def partition(f: IntoArray, *levels: float) -> Sequence[Array]:
   steps = map(subtract, signs[:-1], signs[1:])
   return [.5 - .5 * signs[0]] + [.5 * step for step in steps] + [.5 + .5 * signs[-1]]
 
+def heaviside(f: IntoArray):
+  '''
+  Create a heaviside function based on a levelset:
+  `H(f) = { 0: f < 0; 0.5: f == 0; 1: f > 0`.
```

Comment:
Use double backticks. For coherence with the rest of the module, perhaps you can move the first line with text to the same line as the triple quotes and move the mathematical definition to a separate paragraph, maybe even in a `.. math::` block.